config/.sass-lint.yml

```diff
@@ -0,0 +1,365 @@
+# full list of rules at: https://github.com/sasstools/sass-lint/tree/develop/docs/rules
+options:
+  formatter: stylish
+rules:
+  # Extends
+  extends-before-mixins: 1
+  extends-before-declarations: 1
+  placeholder-in-extend: 1
+
+  # Mixins
+  mixins-before-declarations: 1
+
+  # Line Spacing
+  one-declaration-per-line: 1
+  empty-line-between-blocks: 1
+  single-line-per-selector: 1
+
+  # Disallows
+  no-color-keywords: 1
+  no-color-literals: 2
+  no-css-comments: 2
+  no-debug: 1
+  # Sample - warn against problematic properties
+  # no-disallowed-properties:
+  # - 1
+  # -
+  #  'properties':
+  #    - 'z-index'
+  no-duplicate-properties: 1
+  no-empty-rulesets: 1
+  no-extends: 0
+  no-ids: 2
+  no-important: 1
+  no-invalid-hex: 1
+  no-mergeable-selectors: 1
+  no-misspelled-properties: 1
+  no-qualifying-elements: 1
+  no-trailing-zero: 1
+  no-transition-all: 1
+  no-url-protocols: 1
+  no-vendor-prefixes: 1
+  no-warn: 1
+
+  # Nesting
+  force-attribute-nesting: 1
+  force-element-nesting: 1
+  force-pseudo-nesting: 1
+  bem-depth:
+    - 2
+    -
+      max-depth: 1
```

Comment:
bem-depth: 2 and then max-dept 1? Probably I'm misreading or misunderstanding but that just looks weird, so thought I'd ask.

Reply:
I don't recall - I'll test and check. It does look strange....

Reply:
Ok, I tested and what it's doing is this.

`bem-depth: 2` refers to the setting - 0 is ignore, 1 is warning, 2 is error (preventing the compile)
`max-depth: 1` refers to the number of elements allowed in a BEM selector (the E in BEM). 

I think these are a good place to start. The whole point of BEM is to prevent selector depth, so only allowing a single element seems reasonable to me.

Reply:
Thanks for the clarification. Makes sense!